data/events/2023/unplanned-maintenance-may.mdx

```diff
@@ -2,9 +2,9 @@
 title: Indent x Tailscale Unplanned Maintenance
 date: '2023-05-16'
 displayDate: 'May 16, 2023'
-location: 'San Francisco, CA'
+location: 'San Francisco / London'
 link: 'https://unplannedmaintenance.com/?ref=tsdev'
-displayTime: '5:30 PM – 7:00 PM PST'
+displayTime: '5:30 PM – 7:00 PM Local'
```

Comment:
With the location in both SF and London, what is local referring to?

Reply:
Oh I know this one because I did social for these events! The two in-person events are not simultaneous and are each at their local 5:30pm.

Reply:
Confusing! Thanks